example/Index.re

```diff
@@ -1,5 +1,7 @@
 open Express;
 
+let (>>=) = (f, g, x) => g(f(x));
```

Comment:
Please don't add obtuse operators to code intended to be read by a wider audience. And if you do, at least don't overload an operator that is commonmly used to mean something entirely different. `>>=` usually means `bind`. `>>` is more commonly used for function composition. But again, please avoid using them altogether, since it makes it much harder for newcomers from js to understand what the hell is going on.

Reply:
Will make it less obtuse.